website/docs/components/separator/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,44 @@
+## Usage
+
+### When to use
+
+- Use to visually separate different parts or sections on a page.
+- Use it to create a clear break between paragraphs, blocks, or sections.
+
+### When not to use
+
+- Don’t use to separate headings and body text. Instead, use the proper [Display token](https://github.com/hashicorp/design-system/issues/262) to emphasize hierarchy. 
+
+## Placement
+
+We recommend using meaningful spacing between content elements, allowing users to scan and understand information quickly. Use separators sparingly and purposefully to avoid cluttering the UI and distracting users from the main content.
+
+!!! Do
+
+![Example of a leading icon in the tag](/assets/components/separator/separator-do.png)
+
+!!!
+
+!!! Dont
+
+![Example of a leading icon in the tag](/assets/components/separator/separator-dont.png)
+
+!!!
+
+### Spacing
+
+##### 0px Spacing:
+   - This option works well when you want adjacent components to flow together smoothly without any interruptions in the visual presentation.
+   - Use the 0px spacing option for a tightly integrated layout without any visible gaps between elements.
+   - Avoid using the 0px spacing option when the content sections need clear visual separation or when visual clutter is possible.
+
+##### 24px Spacing:
+   - This option assists in visually distinguishing between different sections, making it simpler for users to navigate and understand the content.
+   - Use the 24px spacing option to create a clear visual distinction and breathing space between content sections.
+   - Similarly, avoid using the 24px spacing option when you require a tightly integrated layout or when limited vertical space is available.
```

Comment:
I'm curious about what you mean by "when limited vertical space is available"...can you clarify? It's a common myth that users don't scroll, so I'm not sure we need to really worry about this. 

```suggestion
   - Avoid using the 24px spacing option when you require a tightly integrated layout or when limited vertical space is available.
```

Reply:
Yeah, looking at again, I see what you mean. I considered when consumers must occupy all the real estate on the screen. So adding that was supposed to communicate compact interfaces / aim for maximizing content within a confined layout.